### PR TITLE
NEWS: Added 1.15.0 section.

### PR DESCRIPTION
**Pull request by rakhmets**
## What
Added 1.15.0 section.

### CODE REVIEW

**Review by yosefe — APPROVED** [review]

**Comment by yosefe** [comment]
@shamisp can you pls review as well?

File: NEWS [hunk 1]
```diff
@@ -104,12 +59,17 @@
 * Added user-side memcpy option for AM benchmarks in ucx_perftest
 * Added wireshark LUA dissectors for some UCX protocols
 #### Build
+* Enabled build with binutils 2.40
```
Comment (shamisp) [inline NEWS:62]: Added support for  binutils 2.40

File: NEWS [hunk 2]
```diff
@@ -104,12 +59,17 @@
...
+* Added versioned dependency to switch between packages with the same names
 * Added a separate xpmem deb subpackage
 * Added aarch64 support to the binary distribution pipeline
 * Removed dependency on libnuma
-
 ### Bugfixes:
 #### UCP
+* Fixed assertion when sending from noncontig GPU buffer to managed buffer
+* Fixed the data race on endpoint configurations
```
Comment (shamisp) [inline NEWS:70]: Fixed the race condition ...

File: NEWS [hunk 3]
```diff
@@ -123,20 +83,29 @@
...
 * Multiple fixes in the new protocol infrastructure information output
 #### UCT
+* Check dmabuf kernel support in ROCm memory domain
 * Fixed exported memh packing
 * Fixed an error in checking return status of multi-threaded memory registration function
 #### RDMA CORE (IB, ROCE, etc.)
+* Fixed dma-buf based memory region registration
+* Fixed memory handle data corruption when PCIe relaxed ordering is enabled
+* Fixed performance degradation when indirect atomic key is not supported by the hardware
+* Fixed remote access error to strict-order key because of wrong offset
```
Comment (shamisp) [inline NEWS:93]: Fixed remote access error to strict-order keys

File: NEWS [hunk 4]
```diff
@@ -123,20 +83,29 @@
 * Fixed CPU/device atomics selection in the new protocol infrastructure
 * Multiple fixes in the new protocol infrastructure information output
 #### UCT
+* Check dmabuf kernel support in ROCm memory domain
```
Comment (shamisp) [inline NEWS:86]: Added check ?

**Review by shamisp — CHANGES_REQUESTED** [review]

File: NEWS [hunk 5]
```diff
@@ -104,12 +59,17 @@
...
+* Added versioned dependency to switch between packages with the same names
 * Added a separate xpmem deb subpackage
 * Added aarch64 support to the binary distribution pipeline
 * Removed dependency on libnuma
-
 ### Bugfixes:
 #### UCP
+* Fixed assertion when sending from non-contiguous GPU buffer to managed buffer
+* Fixed the race condition on endpoint configurations
+* Fixed endpoint reconfiguration issues due to assymetrical selection
```
Comment (tvegas1) [inline NEWS:71]: asymmetrical

**Review by tvegas1 — COMMENTED** [review]
one typo

File: NEWS [hunk 6]
```diff
@@ -75,6 +29,7 @@
 * Added base implementation of is_reachable_v2 API using intra/inter flag
 * Introduced MD capability for non-blocking registration memory types
 #### RDMA CORE (IB, ROCE, etc.)
+* Added format is_reachable_v2 for IB interfaces
```
Comment (shamisp) [inline NEWS:32]: Added implementation of is_reachable_v2  routine to IB interface

**Comment by shamisp** [comment]
Have you updated authors file for this release ?

File: NEWS (hunk 5 above)
Comment (rakhmets) [reply NEWS]: Fixed

File: NEWS (hunk 6 above)
Comment (rakhmets) [reply NEWS]: Fixed.

**Comment by rakhmets** [comment]
> Have you updated authors file for this release ?

Yes, I have. This PRs were merged to v1.15.x https://github.com/openucx/ucx/pull/9250 https://github.com/openucx/ucx/pull/9302

**Review by shamisp — APPROVED** [review]

**Pull request merged**